src/components/decks/DeckDescription.vue

```diff
@@ -0,0 +1,21 @@
+<script>
+import { compile } from 'vue'
+import { parseCardText } from '/src/utils.js'
+
+export default {
+  name: 'DeckDescription',
+  props: {
+    content: {
+      type: String,
+      required: true,
+    },
+  },
+  setup (props) {
+    // Setup accepts a reactive `props` object and can return a render function, so this
+    // functionally allows us to compile arbitrary HTML into Vue components
+    return compile(
+      parseCardText(props.content)
```

Comment:
You'll need to specify all parameters for this to work properly (`ensureParagraphs` makes sure that if there's a single line it's properly wrapped in paragraph tags, and `isLegacy` affects whether embedded card links use legacy cards or Reborn cards).